tools/tcpretrans_example.txt

```diff
@@ -44,6 +44,20 @@ TIME     PID    IP LADDR:LPORT          T> RADDR:RPORT          STATE
 See the "L>" in the "T>" column. These are attempts: the kernel probably
 sent a TLP, but in some cases it might not have been ultimately sent.
 
+To spot heavily retransmitting flows quickly one can use the -c flag. It will
+count occurring retransmits per flow. 
+
+# python tcpretrans.py -c
```

Comment:
For consistency with the other examples, I'd use `./tcpretrans -c` as the command.